devel/google-cloud-sdk/Portfile

```diff
@@ -29,12 +30,14 @@ use_configure       no
 build               {}
 
 destroot {
+    system -W ${worksrcpath} "./install.sh --usage-reporting false --bash-completion false --path-update false --rc-path false --quiet"
```

Comment:
* Why use `--bash-completion` instead of `--command-completion`?
* `--rc-path` doesn't seem to be a true/false argument.

```
usage: install.py [-h] [--usage-reporting USAGE_REPORTING] [--rc-path RC_PATH]
                  [--command-completion COMMAND_COMPLETION]
                  [--path-update PATH_UPDATE] [--disable-installation-options]
                  [--override-components [OVERRIDE_COMPONENTS [OVERRIDE_COMPONENTS ...]]]
                  [--additional-components ADDITIONAL_COMPONENTS [ADDITIONAL_COMPONENTS ...]]
                  [--quiet]

optional arguments:
  -h, --help            show this help message and exit
  --usage-reporting USAGE_REPORTING
                        (true/false) Disable anonymous usage reporting.
  --rc-path RC_PATH     Profile to update with PATH and completion. If given
                        without --command-completion or --path-update in
                        "quiet" mode, a line will be added to this profile for
                        both command completion and path updating.
  --command-completion COMMAND_COMPLETION, --bash-completion COMMAND_COMPLETION
                        (true/false) Add a line for command completion in the
                        profile. In "quiet" mode, if True and you do not
                        provide--rc-path, the default profile will be updated.
  --path-update PATH_UPDATE
                        (true/false) Add a line for path updating in the
                        profile. In "quiet" mode, if True and you do not
                        provide --rc-path, the default profile will be
                        updated.
  --disable-installation-options
                        DEPRECATED. This flag is no longer used.
  --override-components [OVERRIDE_COMPONENTS [OVERRIDE_COMPONENTS ...]]
                        Override the components that would be installed by
                        default and install these instead.
  --additional-components ADDITIONAL_COMPONENTS [ADDITIONAL_COMPONENTS ...]
                        Additional components to install by default. These
                        components will either be added to the default install
                        list, or to the override-components (if provided).
  --quiet, -q           Disable all interactive prompts. If input is required,
                        defaults will be used or an error will be raised
```

Reply:
Thank you for catching those issues. I've fixed them and another one (see commit message).